docs/src/troubleshooting.md

```diff
@@ -0,0 +1,21 @@
+# Troubleshooting and Workflow Tips
+
+## High Failure Rate
+
+While some EKI variants include failure handlers, excessively high failure rates (i.e., > 80%) can lead to inversions finding local minima or failing to converge. To address this:
+
+- **Stabilize the Forward Model**: Ensure the forward model remains stable for small parameter perturbations in offline tests.
```

Comment:
maybe add: "If the forward model is unstable for most of the parameter space, it is challenging to explore it with a calibration method".